opentelekomcloud/services/iam/data_source_opentelekomcloud_identity_project_v3.go

```diff
@@ -107,8 +115,11 @@ func dataSourceIdentityProjectV3Read(_ context.Context, d *schema.ResourceData,
 	}
 
 	if len(filteredProjects) > 1 {
-		log.Printf("[DEBUG] Multiple results found: %#v", allProjects)
-		return fmterr.Errorf("your query returned more than one result")
+		rawProject, err := projects.Get(client, client.ProjectID).Extract()
+		if err != nil {
+			return diag.FromErr(err)
+		}
+		filteredProjects = append(filteredProjects, *rawProject)
```

Comment:
Are you sure it's placed in a correct `if` statement?

Reply:
yes. if will not pass exact project you will get all of your projects, which more then one actually

Reply:
there is a problem unsolved, yes